Makefile

```
@@ -62,19 +70,24 @@ libquirc.so.$(LIB_VERSION): $(LIB_OBJ)
 .c.o:
 	$(CC) $(QUIRC_CFLAGS) -o $@ -c $<
 
+.cxx.o:
+	$(CXX) $(QUIRC_CXXFLAGS) -o $@ -c $<
+
 install: libquirc.a libquirc.so.$(LIB_VERSION) quirc-demo quirc-scanner
 	install -o root -g root -m 0644 lib/quirc.h $(DESTDIR)$(PREFIX)/include
 	install -o root -g root -m 0644 libquirc.a $(DESTDIR)$(PREFIX)/lib
 	install -o root -g root -m 0755 libquirc.so.$(LIB_VERSION) \
 		$(DESTDIR)$(PREFIX)/lib
 	install -o root -g root -m 0755 quirc-demo $(DESTDIR)$(PREFIX)/bin
+	# install -o root -g root -m 0755 quirc-demo-opencv $(DESTDIR)$(PREFIX)/bin
```

We could install if we just tested whether the file exists I believe:
```suggestion
	# test ! -f quirc-demo-opencv || install -o root -g root -m 0755 quirc-demo-opencv $(DESTDIR)$(PREFIX)/bin
```

---

i'm not inclined to do this.
i guess it should be fixed more consistently.
like stop building demos by default as they have different requirements. (opencv, sdl, v4l2)

---

Fair.